Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", ] as String[]
+def paths = []
+
+// using mesos node because it's a lightweight alpine docker image instead of full VM
+node('mesos') {
+  stage("Verify author") {
+    user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#tools-notify')
+  }
+}
+
+node('mesos-ubuntu') {
+  def shcmd = { String command ->
+      output = sh(script: command, returnStdout: true).trim()
+      println(output)
```

Comment:
Why are we printing the output and returning the output? It seems it might be sufficient if do one in the first, and caller prints?

Reply:
We want to print everywhere so it reduces clutter.There is also a spot where we both want to print and return the value